AdaptiveCards/authoring-cards/card-schema.md

```diff
@@ -1,343 +1,383 @@
-   * [`Image`](#schema-image) - The Image Item allows for the inclusion of images in an Adaptive Card.
+   * [`TextBlock`](#schema-textblock) - Displays text allowing control over font sizes, weight, and color.
+   * [`Image`](#schema-image) - Displays images.
+   * [`Media`](#schema-media) - Displays audio or video.
+   * [`MediaSource`](#schema-mediasource) - Defines a source for a Media element
 
 Containers
-   * [`Container`](#schema-container) - Containers group items together
-   * [`ColumnSet`](#schema-columnset) - ColumnSet divides a region into Column's allowing elements to sit side-by-side
-   * [`Column`](#schema-column) - Defines a container that is part of a ColumnSet
-   * [`FactSet`](#schema-factset) - The FactSet Item makes it simple to display a series of facts (e.g. name/value pairs) in a tabular form.
-   * [`Fact`](#schema-fact) - Describes a Fact in a FactSet as a key/value pair
-   * [`ImageSet`](#schema-imageset) - The ImageSet allows for the inclusion of a collection images like a photogallery.
+   * [`Container`](#schema-container) - Containers group items together.
+   * [`ColumnSet`](#schema-columnset) - ColumnSet divides a region into Columns, allowing elements to sit side-by-side.
+   * [`Column`](#schema-column) - Defines a container that is part of a ColumnSet.
+   * [`FactSet`](#schema-factset) - The FactSet element displays a series of facts (i.e. name/value pairs) in a tabular form.
+   * [`Fact`](#schema-fact) - Describes a Fact in a FactSet as a key/value pair.
+   * [`ImageSet`](#schema-imageset) - The ImageSet displays a collection of Images similar to a gallery.
 
 Actions
-   * [`Action.OpenUrl`](#schema-action.openurl) - When Action.OpenUrl is invoked it will show the given url, either by launching it to an external web browser or showing in-situ with embedded web browser.
-   * [`Action.Submit`](#schema-action.submit) - Submit action gathers up input fields, merges with optional data field and generates event to client asking for data to be submitted. It is up to the client to determine how that data is processed. For example: With BotFramework bots the client would send an activity through the messaging medium to the bot.
-   * [`Action.ShowCard`](#schema-action.showcard) - Action.ShowCard defines an inline AdaptiveCard which is shown to the user when it is clicked.
+   * [`Action.OpenUrl`](#schema-action.openurl) - When invoked, show the given url either by launching it in an external web browser or showing in-situ with embedded web browser.
+   * [`Action.Submit`](#schema-action.submit) - Gathers input fields, merges with optional data field, and sends an event to the client. It is up to the client to determine how this data is processed. For example: With BotFramework bots, the client would send an activity through the messaging medium to the bot.
+   * [`Action.ShowCard`](#schema-action.showcard) - Defines an inline AdaptiveCard which is shown to the user when the button or link is clicked.
```

Comment:
Remove "inline". This should simply say "Defines an AdaptiveCard which is shown...". The host gets to choose whether the card is shown inline or as a popup.

Reply:
ah - I can see how this could be confusing. I meant "inline" to mean that the definition of the subcard is inline -- i.e. in the same file. you're right though, better to just strike it :)